--- py-flake8/Makefile.orig	2017-12-01 02:00:49.000000000 +0900
+++ py-flake8/Makefile	2017-12-01 01:58:28.000000000 +0900
@@ -21,4 +21,13 @@
 USE_PYTHON=	autoplist concurrent distutils
 NO_ARCH=	yes
 
-.include <bsd.port.mk>
+.include <bsd.port.pre.mk>
+
+.if ${PYTHON_REL} < 3200
+RUN_DEPENDS+=	${PYTHON_PKGNAMEPREFIX}configparser>0:devel/py-configparser@${FLAVOR}
+.endif
+.if ${PYTHON_REL} < 3400
+RUN_DEPENDS+=	${PYTHON_PKGNAMEPREFIX}enum34>0:devel/py-enum34@${FLAVOR}
+.endif
+
+.include <bsd.port.post.mk>
